introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi! I am Jesús López and I really want to learn how to use git properly. I'm a musician who aims to create a new music learning app.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?